yaml/pois.yaml

```diff
@@ -562,14 +562,34 @@ filters:
       tier: 3
   # university
   - filter: {amenity: university}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.55 ] }, *tier3_min_zoom ] }, 16 ] }
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 10, 10000000 ]
+          - [ 11,  5000000 ]
+          - [ 12,  1000000 ]
+          - [ 13,   500000 ]
+          - [ 14,   500000 ]
```

Comment:
Zoom 14 has same area threshold as zoom 13? Seems different than `tier3_min_zoom` breaks (below), which are also weird.

```
  - &tier3_min_zoom
    lookup:
      key: { col: way_area }
      op: '>='
      table:
        - [ 8,  10000000 ]
        - [ 9,   5000000 ]
        - [ 10,  1000000 ]
        - [ 11,   500000 ]
        - [ 12,   500000 ]
        - [ 13,   100000 ]
        - [ 14,    50000 ]
        - [ 15,     2000 ]
      default: 16
```

Reply:
Comparing tier3 and tier4, I think the following cuts are more consistent with old behavior?

```
- [ 10, 10000000 ]
- [ 11,  1000000 ]
- [ 12,   500000 ]
- [ 13,   100000 ]
- [ 14,    50000 ]
- [ 15,     2000 ]
```

Reply:
Cool, thanks. Updated in 5af8d31.

Yup, the `tier3_min_zoom` was probably where I started from, and the repeated 500,000 was copy pasta from there. As written, it does mean that nothing in tier 3 will get a `min_zoom` of 12. Would we want to change that to 200,000 perhaps?

Reply:
Yes, I'd be fine changing tier3 filter generically to `200,000` as you propose (as well).